promgen/templates/promgen/navbar.html

```diff
@@ -36,7 +36,7 @@
                         <li><a href="{% url 'api:api-root' %}">API</a></li>
                         <li><a href="{% url 'api:all-targets' %}">Export Targets</a></li>
                         <li><a href="{% url 'api:all-rules' %}">Export Rules</a></li>
-                        <li><a href="{% url 'config-urls' %}">Export URL</a></li>
+                        <li><a href="{% url 'api:all-urls' %}">Export URL</a></li>
```

Comment:
I know it's unrelated to your PR, but while at it, would you mind fixing the grammatical number of "URL"?

"Export URL" makes the user think that only one URL will be exported. If you don't mind, could you please change it to "Export URL**s**" (in plural)?

Reply:
Sure, updated and force pushed

Reply:
Re-approved. Thank you very much 👍